docs/Supported-Platforms.md

```diff
@@ -57,7 +57,7 @@ on releases older than Windows 8.1.
 
 Brim's [test automation](#automated-testing) runs on Catalina 10.15 and
 therefore this is the macOS version on which we are best able to ensure quality
-and prevent regressions. Several Brim developers also run macOS Big Sur 11.0
+and prevent regressions. Several Brim developers also run macOS Monterey 12.3
```

Comment:
I can't speak for the official Brim Dev team, but I know when I worked with y'all, folks generally jumped to the new macOS releases as they came out, so here I'm assuming that's still the case.